frontend/lib/search/location_button.dart

```diff
@@ -79,14 +79,8 @@ class _LocationButtonState extends State<LocationButton> {
               await settings.setLocationFeatureEnabled(enabled: true);
             },
           )
-        : await determinePosition(
-            context,
-            requestIfNotGranted: false,
-            onDisableFeature: () async => settings.setLocationFeatureEnabled(enabled: false),
-            onEnableFeature: () async {
```

Comment:
this shouldn't be triggered on every widget rebuild "blindly"
Even if the user deactivates location permission in the OS, the location but does not change to disabled with this implementation.
The user has to click again on the button.
If we want to change this automatically we have to implement it on a proper way

Reply:
Hmm... I think this would only get called once when the widget initializes, right? So that does not seem to be triggering a cycle (at first glance)?

Reply:
well it seem not to be case somehow because it actually caused the rerendering. When i commented out the function in the `WidgetsBinding.instance.addPostFrameCallback` the cycle was gone
Maybe something forces recreation?

Reply:
>  Maybe something forces recreation?

Hmm... it seems like it... That seems to be the root cause, then. Maybe it has something to do with the changes to the routing / deep linking?

Reply:
Yes I also thought about that because it didn't happen before merging that issue. Even I have no idea how the GoRouter should cause an issue which seems not to be related with routing. But maybe sth triggers the app. dart to rerender

Reply:
i reverted the changes. thx for the pr